Vagrantfile

```diff
@@ -138,6 +138,11 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       guest: 7357,
       host: 7357
     }.merge(VAGRANT_NETWORK_OPTIONS)
+    # Livereload
+    app.vm.network "forwarded_port", {
```

Comment:
https://github.com/kdeloach/model-my-watershed/blob/feature/livereload/Vagrantfile#L133
is the same as
https://github.com/kdeloach/model-my-watershed/blob/feature/livereload/Vagrantfile#L143